demo/index.html

```diff
@@ -142,6 +142,49 @@
     }());
   </script>
 
+  <p>Here's a pie chart listening for `onmouseover`:</p>
+
+  <style>
+    #mouseover-demo {
+      position: relative;
+      height: 300px;
+    }
+
+    #mouseover-chart {
+      float: left;
+    }
+
+    #mouseover-display {
+      display: inline-block;
+      position: relative;
+      top: 50%;
+    }
+  </style>
+  <div id="mouseover-demo">
+    <google-chart
+      id="mouseover-chart"
+      type="pie"
+      options='{"title": "Distribution of days in 2001H1"}'
+      cols='[{"label": "Month", "type": "string"},{"label": "Days", "type": "number"}]'
+      rows='[ ["Jan", 31],["Feb", 28],["Mar", 31],["Apr", 30],["May", 31],["Jun", 30] ]'
+      events='["onmouseover"]'>
```

Comment:
Searching the code, it looks around 30. The events vary pretty wildly from chart to chart, too:
- `regionClick`, `zoomOut`, `drawingDone` look to be GeoMap specific
- `page` for Tables
- `uichange`, `statechange` just for control types
- etc.
I'd be all for attaching all events like maps does if you think that's better.
I was mainly afraid that it seems like eventNames are arbitrary and an implemented visualization can use anything it wants. But I guess we can always rectify issues with that.
We could extend the chart metadata to include the list of supported events and use that to attach only the events associated with a chart. Would be more overhead on the library, but probably the best option, IMO.
e.g.:
```js
{
  type: 'table',
  ctor: 'Table',
  events: ['page']
}
```